BUILD.bazel

```diff
@@ -59,6 +59,7 @@ android_binary(
         "versionName": "0.1-alpha",
     },
     multidex = "native",  # TODO(#1875): Re-enable legacy for optimized release builds.
+    visibility = ["//visibility:public"],
```

Comment:
I'm not sure what package_group I need to use here

Reply:
We ought to create another binary specifically for tests, set that visibility to oppia_testing_visibility & set is as testonly.

Reply:
Done